solutions/kueue-admission-check/env/authtokenrequest-c1.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: multicluster.x-k8s.io/v1alpha1
+kind: AuthTokenRequest
```

Comment:
I would not think we should have this, since this is not decided yet.

Reply:
yes, let's replace it with clusterpermission and managedserviceaccount.